tests/testthat/test_01-checkFixCfg.R

```diff
@@ -7,8 +7,7 @@
 test_that("checkFixCfg works", {
   remind_folder <- "../.."
   savecfg <- cfg <- gms::readDefaultConfig(remind_folder)
-  # regexp = NA means: no warning
-  expect_warning(checkFixCfg(cfg, remind_folder, testmode = TRUE), regexp = NA)
+  expect_no_warning(checkFixCfg(cfg, remind_folder, testmode = TRUE))
```

Comment:
Wait, what? Did not know that existed, I hated the `regexp = NA` stuff. Turns out it was [just introduced](https://cran.r-project.org/web/packages/testthat/news/news.html) in version 3.1.5 as "experimental"… But if it works here, that is fine.

But you could think about adding `testthat (>= 3.1.5)` as dependency in `DESCRIPTION`, just to be safe.